scripts/train.py

```diff
@@ -0,0 +1,44 @@
+import os
+import click
+import pickle
+import sys
+from sklearn.neural_network import MLPClassifier
+from pkspace.utils.loaders import PKSpaceLoader, PKLotLoader
+from pkspace.utils import trainer
+
+
+@click.command()
+@click.option('--loader', '-l', type=click.Choice(['PKLot', 'PKSpace']),
+              default='PKSpace', help='Loader used to load dataset')
+@click.option('--model_type', '-mt', type=click.Choice(['MLP']), default='MLP',
+              help='Type of model to be trained')
+@click.option('--model_path', '-pm', type=click.Path(exists=True),
+              default=None,
+              help='Path to trained model, to be used as a base in training')
+@click.argument('dataset_dir', type=click.Path(exists=True))
+@click.option('--output', '-o', default='out.pkl',
+              help='Path to output file for trained model')
+def train(loader, model_type, model_path, dataset_dir, output):
+    if not os.path.isdir(dataset_dir):
+        sys.stderr.write('{} is not a directory'.format(dataset_dir))
+        sys.exit(1)
+
+    if loader == 'PKSpace':
+        loader = PKSpaceLoader()
+    elif loader == 'PKLot':
+        loader = PKLotLoader()
+
+    if model_path is not None:
+        with open(model_path, 'rb') as fp:
+            model = pickle.load(fp)
+    elif model_type == 'MLP':
+        model = MLPClassifier(solver='lbfgs', hidden_layer_sizes=(15, 10))
```

Comment:
@rstevanak The problem here is that the nargs can't be set to > 2 or something like that. You can try to do it with `multiple` flag where user would specify number of neurons for every layer, for example, `-h 15 -h 10 -h 50` where this would correspond to 3 layer network. The problem here is that there might be sometime in the future more models that do not have to have layers so I think it might be best to just let user specify some values in some `json` file or `yml` or something I do not really know a better way. @mrshu what are your thoughts?